CONTRIBUTING.md

```diff
@@ -237,18 +237,18 @@ Please note that if adding a runtime guarded feature, your [release notes](chang
 * Typically we try to turn around reviews within one business day.
 * See [OWNERS.md](OWNERS.md) for the current list of maintainers.
 * It is generally expected that a senior maintainer should review every PR to
-  core code. Test-only or extension-only changes need only be reviewed by a
+  core code. Test-only or extension-only, or tool-only changes need only be reviewed by a
   maintainer, or senior extension maintainer.
 * It is also generally expected that a "domain expert" for the code the PR touches should review the
   PR. This person does not necessarily need to have commit access.
-* The previous two points generally mean that every PR should have two approvals. (Exceptions can
```

Comment:
I think we regularly end up with a cross-company domain review by a senior maintainer so I'm not sure this one really applies.